### ISSUE TITLE
JetEngine 3.5.0 - Fatal Error 

### ISSUE THREAD

**Issue by rtpHarry** [CrocoBlock/suggestions#7597]
Just upgraded a site to 3.5.0 and the listing grids all had fatal errors, in front end view and in elementor.

Not sure what it is as it doesn't say which property / object its failing on, but here is the error_log output:

```
[01-Jul-2024 15:04:23 UTC] PHP Fatal error:  Uncaught Error: Call to a member function get_prop_default() on false in /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/elementor-views/components/component-document.php:246
Stack trace:
#0 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/elementor-views/components/component-document.php(305): Jet_Engine\Elementor_Views\Components\Document->get_default_component_props()
#1 /home/sitenamehere/public_html/wp-content/plugins/elementor/core/base/document.php(1192): Jet_Engine\Elementor_Views\Components\Document->get_container_attributes()
#2 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper()
#3 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/elementor-views/frontend.php(172): Elementor\Frontend->get_builder_content()
#4 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/elementor-views/frontend.php(131): Jet_Engine_Elementor_Frontend->get_listing_content()
#5 /home/sitenamehere/public_html/wp-includes/class-wp-hook.php(324): Jet_Engine_Elementor_Frontend->get_listing_content_cb()
#6 /home/sitenamehere/public_html/wp-includes/plugin.php(205): WP_Hook->apply_filters()
#7 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/frontend.php(297): apply_filters()
#8 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/frontend.php(273): Jet_Engine_Frontend->get_listing_item_content()
#9 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/render/listing-grid.php(1538): Jet_Engine_Frontend->get_listing_item()
#10 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/render/listing-grid.php(1421): Jet_Engine_Render_Listing_Grid->posts_loop()
#11 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/render/listing-grid.php(992): Jet_Engine_Render_Listing_Grid->posts_template()
#12 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/render/listing-grid.php(83): Jet_Engine_Render_Listing_Grid->render_posts()
#13 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/render/base.php(240): Jet_Engine_Render_Listing_Grid->render()
#14 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/listings/manager.php(467): Jet_Engine_Render_Base->render_content()
#15 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/elementor-views/static-widgets/grid.php(2789): Jet_Engine_Listings->render_listing()
#16 /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/elementor-views/static-widgets/grid.php(2793): Elementor\Jet_Listing_Grid_Widget->render_posts()
#17 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(2325): Elementor\Jet_Listing_Grid_Widget->render()
#18 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/widget-base.php(635): Elementor\Controls_Stack->render_by_mode()
#19 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/widget-base.php(776): Elementor\Widget_Base->render_content()
#20 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Widget_Base->print_content()
#21 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#22 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#23 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#24 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#25 /home/sitenamehere/public_html/wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#26 /home/sitenamehere/public_html/wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements()
#27 /home/sitenamehere/public_html/wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements()
#28 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper()
#29 /home/sitenamehere/public_html/wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content()
#30 /home/sitenamehere/public_html/wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content()
#31 /home/sitenamehere/public_html/wp-includes/plugin.php(205): WP_Hook->apply_filters()
#32 /home/sitenamehere/public_html/wp-includes/post-template.php(256): apply_filters()
#33 /home/sitenamehere/public_html/wp-content/plugins/elementor/modules/page-templates/module.php(190): the_content()
#34 [internal function]: Elementor\Modules\PageTemplates\Module->print_callback()
#35 /home/sitenamehere/public_html/wp-content/plugins/elementor/modules/page-templates/module.php(207): call_user_func()
#36 /home/sitenamehere/public_html/wp-content/plugins/elementor/modules/page-templates/templates/header-footer.php(19): Elementor\Modules\PageTemplates\Module->print_content()
#37 /home/sitenamehere/public_html/wp-includes/template-loader.php(106): include('/home/sitenamehere/...')
#38 /home/sitenamehere/public_html/wp-blog-header.php(19): require_once('/home/sitenamehere/...')
#39 /home/sitenamehere/public_html/index.php(17): require('/home/sitenamehere/...')
#40 {main}
  thrown in /home/sitenamehere/public_html/wp-content/plugins/jet-engine/includes/components/elementor-views/components/component-document.php on line 246
```

Rolling back to 3.4.6 has got the site up and running again.

**Comment by alanstainerPMW** [comment]
I have found the exact same problem with JetEngine 3.5.0 Rolling back to an earlier version got things back up and running again.

**Comment by Crocoblock** [comment]
hi @rtpHarry @alanstainerPMW 

this has been fixed and released today in v3.5.1

Thank you for your report!

**Comment by rtpHarry** [comment]
confirmed that this has resolved my issue, thanks
